### DISCUSSION TITLE
Trouble converting my OUTCAR to deepmd/npy (example/data_conv OUTCAR works)

### DISCUSSION THREAD

**Discussion by kyladevilla** [deepmodeling/dpdata#434] [Q&A] [answered]
Hello, 
I recently upgraded to DeePMD v2.2.0. 
When converting the sample OUTCAR (~/deepmd-kit-2.2.0/examples/data_conv/OUTCAR) to deepmd_data format I have no issues. I'm using the script given in the manual:

![image](https://user-images.githubusercontent.com/99236935/226447387-449fcac0-58dc-45bc-b69c-57d77d475187.png)

However, when trying to convert my own OUTCAR files, I am met with the following error: 
![image](https://user-images.githubusercontent.com/99236935/226447977-9910c9fb-edd2-459b-b157-7bc078185b34.png)

I have tried:
OUTCARs of 2 or 3 elements. --> no change
OUTCARs with only 1 ionic step --> no change
adding "fmt="vasp/outcar" to end of first line --> no change 

I am not sure what is different in my OUTCAR files that is causing this issue. The version of VASP used is the same (vasp.5.4.4.18Apr17). 
[OUTCAR.zip](https://github.com/deepmodeling/deepmd-kit/files/11022413/OUTCAR.zip)

I've attached an example OUTCAR file that will not work for me, I would appreciate any advice.

**Comment by wanghan-iapcm** [comment, marked as answer]
This should be a issue of `dpdata`. Which version are you using?

**Reply by kyladevilla** [reply]
Thanks for the help. I am using dpdata v0.2.14

**Reply by kyladevilla** [reply]
If it helps, I went into /deepmd/lib/python3.10/site-packages/dpdata/utils.py and recorded some print statements. 
"atom_names" is empty, but "atom_types" is not: 
![image](https://user-images.githubusercontent.com/99236935/226504856-f73472f7-bb6b-401d-8e2c-7c729750ab2b.png)

**Reply by kyladevilla** [reply]
If I go into /deepmd/lib/python3.10/site-packages/dpdata/utils.py and manually define "atom_names" the code works fine and I get the deepmd_data folder. I'm just not sure where the error is in filling in the "atom_names". 

![image](https://user-images.githubusercontent.com/99236935/226506329-9e2b9b59-931d-44d3-bade-6d1cd115e5a6.png)

**Reply by wanghan-iapcm** [reply]
The section of  information of POTCAR is missing in our OUTCAR. By default it should be printed. 
An example of the section is the following 
```
   VRHFIN =Al: s2p1                                                             
   LEXCH  = PE                                                                  
   EATOM  =    53.5387 eV,    3.9350 Ry                                         
                                                                                
   TITEL  = PAW_PBE Al 04Jan2001                                                
   LULTRA =        F    use ultrasoft PP ?                                      
   IUNSCR =        1    unscreen: 0-lin 1-nonlin 2-no                           
   RPACOR =    1.500    partial core radius                                     
   POMASS =   26.982; ZVAL   =    3.000    mass and valenz                      
   RCORE  =    1.900    outmost cutoff radius                                   
   RWIGS  =    2.650; RWIGS  =    1.402    wigner-seitz radius (au A)           
   ENMAX  =  240.300; ENMIN  =  180.225 eV                                      
   ICORE  =        2    local potential                                         
   LCOR   =        T    correct aug charges                                     
   LPAW   =        T    paw PP                                                  
   EAUG   =  291.052                                                            
   DEXC   =    -.041                                                            
   RMAX   =    2.974    core radius for proj-oper                               
   RAUG   =    1.300    factor for augmentation sphere                          
   RDEP   =    1.966    radius for radial grids                                 
   QCUT   =   -4.203; QGAM   =    8.405    optimization parameters              
```

**Reply by kyladevilla** [reply]
Adding that part from the POTCAR files to the top of my OUTCAR fixed the problem! Thank you so much for helping me with this!
